Doc/library/calendar.rst

```diff
@@ -138,6 +138,11 @@ interpreted as prescribed by the ISO 8601 standard.  Year 0 is 1 BC, year -1 is
 
    :class:`TextCalendar` instances have the following methods:
 
+   .. method:: prweek(theweek, width)
+
+      Print a week's calendar as returned by :meth:`formatweek` and sets the
```

Comment:
This will silence the warning (it can be reverted when the `formatweek` documentation is added):
```suggestion
      Print a week's calendar as returned by :meth:`!formatweek` and sets the
```